src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkQuantumStorage.java

```diff
@@ -88,6 +91,14 @@ public class NetworkQuantumStorage extends SlimefunItem implements DistinctiveIt
         Theme.PASSIVE + "Shift Click to change voiding"
     );
 
+    private static final ItemStack SET_ITEM_SUPPORTING_CUSTOM_MAX = new CustomItemStack(
+        Material.LIME_STAINED_GLASS_PANE,
+        Theme.SUCCESS + "Set Item",
+        Theme.PASSIVE + "Drag an item on top of this pane to register it.",
+        Theme.PASSIVE + "Shift Click to change voiding",
+        Theme.PASSIVE + "Click with no item to set maximum storage size"
```

Comment:
I would flip the order of shift click and click with no item. As well as potentially reword "drag an item" for consistency and organization

Reply:
Do you think it would be better worded as:
"Click with an item to set the Quantum's Stored Item"
"Click with no item to set the Quantum's maximum storage size"
"Shift Click to toggle voiding items over the maximum storage size"

Reply:
I'd do "Click without an item(...)" but otherwise yes, I'd also like to hear what sefi thinks on this